frontend/src/atom/textAtom.ts

```diff
@@ -4,15 +4,15 @@ import { atom, atomFamily } from 'recoil';
 export const TEXT_WHITE: RGBColor = { r: 251, g: 251, b: 251, a: 1 };
 export const TEXT_BLACK: RGBColor = { r: 10, g: 10, b: 10, a: 1 };
 
-export const TEXT_LG = 'text-lg';
-export const TEXT_MD = 'text-md2';
-export const TEXT_SM = 'text-sm';
-export type TextSizeType = 'text-lg' | 'text-md1' | 'text-md2' | 'text-sm' | 'text-xsm';
+export const TEXT_LG = 'tablet:text-[34px] text-[21px]';
+export const TEXT_MD = 'tablet:text-[26px] text-[16px]';
+export const TEXT_SM = 'tablet:text-[22px] text-[14px]';
+export type TextSizeType = string;
 
 export type TextType = {
   id: number;
   content: string;
-  fontSize: TextSizeType;
+  fontSize: string;
```

Comment:
```suggestion
  fontSize: 'tablet:text-[34px] text-[21px]' | 'tablet:text-[26px] text-[16px]' | 'tablet:text-[22px] text-[14px]';
```
줄이 길어져서 type을 한번 선언하고 재반영한 코드라, string으로 수정하면, 차라리 위쪽 type TextSizeType을 지우고 이부분에 작성해도 될 것 같아요. string 타입은 사용자에게 입력을 받거나, 문자열 자체의 타입때만 사용하는게 좋고, enum class 만들듯이 일정한 속성이 정해져 있다면 고정값을 직접 타입에 주는 것도 좋을 것 같습니다.

Reply:
제안 미반영하고 resolve 하셔도 됩니다!